src/style.css

```diff
@@ -1,3 +1,111 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+h1 {
+  margin: 20px;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__items {
+  display: block;
+}
+
+.logo {
+  display: block;
+  padding: 10px 0;
+  height: 40px;
+}
+
+.nav__link {
+  display: block;
+  margin: 0 0 0 20px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  height: 60px;
+  line-height: 60px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+a:hover {
+  color: #00acdc;
+  text-decoration-color: #00acdc;
```

Comment:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)